playwright-e2e/pages/patient-type.ts

```diff
@@ -0,0 +1,71 @@
+interface CancerDiagnosed {
+  prompt: string;
+  typeCancer: string;
+}
+interface PatientData {
+  whoIsSigningUp: string;
+  cancerDiagnosed: CancerDiagnosed;
+  researchContentForm: string;
+  ddpTestID: {
+    firstName: string;
+    lastName: string;
+    signature: string;
+    authorizationSignature: string;
+    dateOfBirthLocation: string;
+    relationshipChild?: string;
+    assentFirstName?: string;
+    assentLastName?: string;
+    assentSignature?: string;
+    assentChildSignature?: string;
+    bloodSamples?: string;
+    assentTissue?: string;
+  };
+}
+
+export type TypePatient = 'child' | 'adult';
+
+export type Patient = {
+  [index in TypePatient]: PatientData;
+};
+
+export const PatientsData: Patient = {
+  child: {
+    whoIsSigningUp: 'My child has been diagnosed with cancer(s) and I am signing up for them or with them.',
+    cancerDiagnosed: {
+      prompt: 'What primary cancer(s) has your child been diagnosed with? ',
+      typeCancer: 'Leukemia'
+    },
+    researchContentForm: 'Research Consent Form - Parent or Guardian',
+    ddpTestID: {
```

Comment:
So happy to see this selectors--we had instrumented this stuff years ago and now it's being used just as we imagined :)